From e0fc5bea466c16b9d754a8ac8d0453e396678a2c Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 19 Feb 2014 20:28:07 -0800
Subject: [PATCH] KERNEL_INC in modern kernel should point at toplevel
 kerneldir

Reason is kernel headers are not only in toplevel
include/ but also in toplevel arch/ dir

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 031d8bc..73e9b3c 100644
--- a/Makefile
+++ b/Makefile
@@ -245,15 +245,15 @@ endif
 
 modules_edma:
 	@echo "Going to compile edma test kernel modules for $(PLATFORM)"
-	cd testcases/ddt/edma_test_suite/src/kernel; make CROSS_COMPILE=$(CROSS_COMPILE) KERNEL_DIR=$(KERNEL_INC)/.. PLATFORM=$(PLATFORM)
+	cd testcases/ddt/edma_test_suite/src/kernel; make CROSS_COMPILE=$(CROSS_COMPILE) KERNEL_DIR=$(KERNEL_INC) PLATFORM=$(PLATFORM)
 
 modules_gpio:
 	@echo "Going to compile gpio test kernel modules for $(PLATFORM)"
-	cd testcases/ddt/gpio_test_suite/src/kernel; make CROSS_COMPILE=$(CROSS_COMPILE) KERNEL_DIR=$(KERNEL_INC)/.. PLATFORM=$(PLATFORM)
+	cd testcases/ddt/gpio_test_suite/src/kernel; make CROSS_COMPILE=$(CROSS_COMPILE) KERNEL_DIR=$(KERNEL_INC) PLATFORM=$(PLATFORM)
 
 modules_ipc:
 	@echo "Going to compile IPC test kernel modules for $(PLATFORM)"
-	cd testcases/ddt/ipc_test_suite/src/kernel; make CROSS_COMPILE=$(CROSS_COMPILE) KERNEL_DIR=$(KERNEL_INC)/.. PLATFORM=$(PLATFORM)
+	cd testcases/ddt/ipc_test_suite/src/kernel; make CROSS_COMPILE=$(CROSS_COMPILE) KERNEL_DIR=$(KERNEL_INC) PLATFORM=$(PLATFORM)
 
 modules: $(MODULES_TO_BUILD)
 
-- 
1.9.0

